src/compute/impl/NextPrimeCompute.java

```diff
@@ -25,7 +25,7 @@ public long compute(int i) {
 			}
 		return isPrimeImproved(i) ? recursiveCompute(i + 1) : recursiveCompute(i);
 	}
-	public static boolean isInteger(Object obj) { 
+	boolean isInteger(Object obj) { 
 		return obj instanceof Integer; 
```

Comment:
Note that this check will fail if the thing passed isn't an object.
Since the int type is a primitive and not an object, this check will never pass.
`Instanceof` only checks for inheritance-based relationships.